autosubmit/platforms/paramiko_platform.py

```diff
@@ -750,6 +749,39 @@ def job_is_over_wallclock(self, job, job_status, cancel=False):
                     Log.debug(f"Error cancelling job {job.id}: {str(e)}")
         return job_status
 
+    def get_completed_job_names(self, job_names_provided: Optional[list[str]] = None) -> list[str]:
+        """Retrieve the names of all files ending with '_COMPLETED' from the remote log directory using SSH.
+
+        :param job_names_provided: If provided, filters the results to include only these job names.
+        :type job_names_provided: Optional[List[str]]
+        :return: List of job names with COMPLETED files.
+        :rtype: List[str]
+        """
+        job_names = []
+        if self.expid in str(self.remote_log_dir):  # Ensure we are in the right experiment
```

Comment:
Can that really happen? If so, maybe it'd be good to log (I think if it happens it is not expected or intentional?)

```
        if not self.expid in str(self.remote_log_dir):
            Log.warning(f'Trying to locate _COMPLETED files in the wrong expid remote log dir {self.remote_log_dir}')
```